test/evergreen.yml

```diff
@@ -821,6 +839,16 @@ variables:
     ENABLE_SHARED: -DENABLE_SHARED=0
     ENABLE_STATIC: -DENABLE_STATIC=1
 
+  # Configure flags for tiered storage Azure extension.
```

Comment:
It seems that each variable that is defined there is a comment that comes along with it. I agree that the comment is redundant but because of consistency I added it here, do you think we should remove these? S3 also does the same.

Reply:
I think the redundancy is fine, it's good to know the purpose of the next few lines with a small comment.

Reply:
Sure - looks like I've been outvoted here :-)